WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -346,4 +349,38 @@ + (NSRange)bindingPortRangeFromArguments
   return NSMakeRange(port, 1);
 }
 
++ (void)setReduceMotionEnabled:(BOOL)isEnabled
+{
+  if (isSDKVersionGreaterThanOrEqualTo(@"10.0")) {
+    // setReduceMotionEnabled exists over iOS 10
+    return;
+  }
+
+  void *handle = dlopen(settingsPrefBundlePath, RTLD_LAZY);
```

Comment:
should we check if the dlib has been loaded successfully?

Reply:
Will remove.
I tested like `setValue:value forPreferenceKey:FBKeyboardPredictionKey` like keyboard, but I could not find the proper key for this reduceMotion.